src/main/webapp/js/user-page-loader.js

```diff
@@ -127,11 +141,14 @@ function fetchBlobstoreUrlAndShowForm() {
     });
 }
 
+function redirectToNewPost() {
+  location.href = "new-post.html";
+}
+
 /** Fetches data and populates the UI of the page. */
 function buildUI() {
   setPageTitle();
   showMessageFormIfViewingSelf();
-  createMapForUserPage();
   fetchMessages();
   fetchAboutMe();
   fetchBlobstoreUrlAndShowForm();
```

Comment:
`loadMarkdownEditor()` could possibly be removed, its causing a console message to appear on `user-page` because there is no more text field to edit